src/PlotlyPanel.tsx

```diff
@@ -65,7 +70,7 @@ export const PlotlyPanel: React.FC<Props> = (props) => {
         fill: options.series.areaFill && options.series.plotType === 'line' ? 'tozeroy' : 'none',
         marker: {
           size: options.series.markerSize,
-          color: getFieldSeriesColor(yField, theme).color,
+          color: getFixedColor(yField, theme)
```

Comment:
`getFieldSeriesColor` was skipping colors in some cases. It was a minor issue, but it's easier to just feed the Grafana color palette into Plotly using the `colorway` setting below and let Plotly pick automatically.
Then all we need to do here is handle the case where the user explicitly overrides a field's color.